Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -801,6 +801,8 @@
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks";
 				MACOSX_DEPLOYMENT_TARGET = 13.4;
 				OTHER_LDFLAGS = "-ObjC";
+				"PRODUCT_BUNDLE_IDENTIFIER[sdk=iphoneos*]" = "com.andrewmuniz.Basic-Car-Maintenance";
+				"PRODUCT_BUNDLE_IDENTIFIER[sdk=macosx*]" = "com.andrewmuniz.Basic-Car-Maintenance";
```

Comment:
Don't commit these! you can edit the `pbxproj` file manually in a text editor like VSCode (make sure Xcode is _not_ open when you do this

Reply:
So you just want me to 'git ignore' this file next time I commit new changes to the branch so it does not push up with the changes?

Reply:
@am2089 **NO!!** the pbxproj must be there, this is basically what makes the xcodeproj file. You need to manually use a text editor and fix these parts that were added, because they shouldn't have been added